packages/at_onboarding_cli/pubspec.yaml

```diff
@@ -6,35 +6,39 @@ homepage: https://atsign.com
 documentation: https://docs.atsign.com/
 
 environment:
-  sdk: '>=3.3.0 <4.0.0'
+  sdk: ^3.6.0
+resolution: workspace
 
 executables:
   at_register: register_cli
   at_activate: activate_cli
 
 dependencies:
-   args: ^2.5.0
-   alfred: ^1.1.2+1
-   crypton: ^2.2.1
-   encrypt: ^5.0.3
-   http: ^1.2.1
-   image: ^4.1.7
-   meta: ^1.14.0
-   path: ^1.9.0
-   zxing2: ^0.2.0
-   at_auth: ^2.3.0
-   at_chops: ^2.2.0
-   at_client: ^3.5.0
-   at_commons: ^5.0.2
-   at_lookup: ^3.0.52
-   at_server_status: ^1.0.5
-   at_utils: ^3.2.0
-   duration: ^4.0.3
-   crypto: ^3.0.5
-   chalkdart: ^2.0.9
+  # TODO review this for dependencies only used in tests which should be
+  # under dev_dependencies. alfred was in the wrong spot, which indicates that
+  # others may be too.
```

Comment:
I found a dependency that should have been under dev_dependencies and moved it (alfred). It looks like some of these others may be the same. More main-line dependencies increases the chances that a consumer of the package experiences a version constraint conflict.